plugins/out_splunk/splunk.c

```diff
@@ -628,7 +628,6 @@ static void cb_splunk_flush(struct flb_event_chunk *event_chunk,
     flb_sds_t buf_data;
     size_t resp_size;
     size_t buf_size;
-    char *endpoint;
```

Comment:
While looking through the file for potential other uses of the `endpoint` defined in L690-696, I found this one as well. Unless I miss something, this is never used anywhere, so guess this is an artifact of an earlier implementation. Given that this PR removes the ability to switch endpoints, I removed the variable as well. LMK if you want me to roll this back.

Reply:
My mistake, this PR removes the only occurrences and thus the removal is solid.